### ISSUE TITLE
Lesson 4, ensure the bin directory of yarn is in your path before running ganache-cli

### ISSUE THREAD

**Issue by fantauzzi** [smartcontractkit/full-blockchain-solidity-course-py#458]
After installing `ganache-cli` with yarn, before running it you should make sure the `bin` directory of  yarn is in your path. If it is the first time you are installing anything with yarn, that is likely not the case yet. Under Linux, the directory to be added to your PATH is (starting from your home) `.yarn/bin` .

**Comment by cromewar** [comment]
Thanks for the advice @fantauzzi may be we could add this to troubleshot page in order more people take this in consideration.

**Comment by SimonSvis** [comment]
Hi @fantauzzi , I believe I encountered an issue that I could solve with your help. When I try to check the ganachi version VSC says the term isn't recognized. Could you please tell me how to do I add Yarn it to my PATH? I'm a Windows user by the way.

**Comment by cromewar** [comment]
@SimonSvis does this help you? 
https://stackoverflow.com/questions/9587665/nodejs-cannot-find-installed-module-on-windows

**Comment by SimonSvis** [comment]
@cromewar Thanks for reaching out crome! I tried the solutions that you linked, but I didn't manage to solve it that way. [This](https://stackoverflow.com/questions/69236378/error-in-ganache-installation-the-term-ganache-cli-is-not-recognized-as-the-n) on the other hand solved my issue. It seemed like I installed it properly with yarn(according to the video), but I couldn't run ganache, so perhaps it wasn't installed properly after all.
